src/HOHQMesh.jl

```diff
@@ -223,10 +230,11 @@ function generate_mesh(control_file;
     flush(tmpio)
 
     # Run HOHQMesh and store output
+    str_subdivision_maximum = string(subdivision_maximum)
```

Comment:
I think there's no need for a manual conversion
```suggestion
```

Reply:
You are right, I can directly provide `subdivision_maximum` and it still works. I will update the commands accordingly.